src/shared/seccomp-util.c

```diff
@@ -1619,6 +1620,12 @@ int seccomp_memory_deny_write_execute(void) {
                         return r;
                 if (r < 0)
                         log_debug_errno(r, "Failed to install MemoryDenyWriteExecute= rule for architecture %s, skipping: %m", seccomp_arch_to_string(arch));
+                loaded++;
+        }
+
+        if (loaded == 0) {
+            log_debug_errno(r, "Failed to install any seccomp rules for MemoryDenyWriteExecute=");
+            return 1;
```

Comment:
We prefer to return 1if something happened, and 0 if nothing did. Also 8 space indentation. Maybe just do:
```C
if (loaded == 0)
            log_debug_errno(r, "Failed to install any seccomp rules for MemoryDenyWriteExecute=");
return loaded;
```

This return value is not used by anything? I'm not sure what you are trying to do here.

Reply:
Thanks for taking a look @keszybz
While src/core/execute.c doesn't care about the retval at least the tests in src/test/test-seccomp.c would and that way could spot issues like the main issue the first commit tries to address.

The already existing semantic of the retval is following `seccomp_rule_*` which is to "return zero on success, negative errno values on failure". Your suggestion would still into that as a positive value (the rules loaded) would match well that pattern. So I'm adopting your suggestion.

Also I fixed the 8 space indent, thanks for the hint.

This will be in a coming push to the branch, but I see that @poettering just added review feedback as well. I'll go through that first before pushing anything.